tanner/utils/db_helper.py

```diff
@@ -54,14 +48,13 @@ def insert_dummy_data(table_name, data_tokens, cursor):
                 if token == 'I':
                     values.append(i)
                 if token == 'L':
-                    data = random.choice(dummy_data)
+                    data = elizabeth.Personal().full_name()
```

Comment:
L is for login/username data, maybe `elizabeth.Personal().username()` more suitable her?